src/components/components/ebay-dialog-base/migrator.js

```diff
@@ -0,0 +1,43 @@
+const headerRegExp = /^h[1-6]$/;
+
+/**
+ * @description
+ * Changes @heading content if wrapped with <h[1-6]> to be unrapped
+ */
+
+function migrator(el, context) {
+    const walker = context.createWalker({
+        enter(node) {
+            if (node.tagName === '@header') {
+                // Iterate only once in caes there are multiple tags
+                let foundH2 = false;
+                node.forEachChild((child) => {
+                    const tag = child.tagName;
+                    if (!foundH2 && headerRegExp.test(tag)) {
+                        foundH2 = true;
+                        const isDefault = tag === 'h2';
+                        const message = isDefault ? `<@header>` : `<@header as=${tag}>`;
```

Comment:
`tag` here needs to be quoted. Currently would output `<@header as=h1>` which is not valid Marko.

Reply:
I guess this is just for the message, but still.